core/google/cloud/_helpers.py

```diff
@@ -34,7 +34,7 @@
     import grpc
     from google.auth.transport.grpc import (
         AuthMetadataPlugin)  # pragma: NO COVER
-except ImportError:
+except ImportError:  # pragma: NO COVER
```

Comment:
It is in one of the several environments you run coverage in, but not in another.